app/serializers/order_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class OrderSerializer < ActiveModel::Serializer
+  attributes :id, :total, :created_at, :updated_at
+
+  has_many :line_items
+end
```

Comment:
For our purposes in this assessment, this serialization class was added to allow for the line items to  be included in the orders payload via the orders api endpoint.